<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarCliente</class>
 <widget class="QDialog" name="EditarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Cliente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnSalvarEdicaoCliente">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>   Salvar</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/Imagens/Imagens/salvar.png</normaloff>:/Imagens/Imagens/salvar.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelarEdicaoCliente">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>   Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/Imagens/Imagens/cancelar.png</normaloff>:/Imagens/Imagens/cancelar.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblEditarCliente_ID">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarCliente_CPF">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarCliente_Nome">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Endere√ßo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarCliente_Endereco">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Data de Nascimento:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarCliente_DataNascimento">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarCliente_Telefone">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarCliente_Email">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
